pkg/login/social/gitlab_oauth.go

```diff
@@ -51,6 +51,11 @@ func (s *SocialGitlab) IsGroupMember(groups []string) bool {
 	return false
 }
 
+func (s *SocialGitlab) OrgToRoleMap() map[string][]SocialGroup {
+  return nil
+}
+
+
```

Comment:
I think we should change other oauth providers if not needed in order to keep the patch as small as possible and make it easier to merge upstream changes

Reply:
@richerve Correct me if i'm wrong, but I think you're proposing *not* adding these methods? If so, that is not possible as the current implementation adds this method signature to the [underlying interface](https://github.com/mongodb-forks/grafana/blob/f547222f0733d035778f13767ae4ff6202cbcd43/pkg/login/social/social.go#L43). So all implementations of that interface needs to implement this method to satisfy the interface and successfully compile.